docs/pages/linux-demo.mdx

```diff
@@ -101,67 +172,163 @@ $ curl (=teleport.teleport_install_script_url=) | bash -s (=teleport.version=)
 Generate a configuration file for Teleport using the `teleport configure` command.
 This command requires information about a TLS certificate and private key.
 
-(!docs/pages/includes/tls-certificate-setup.mdx!)
+The instructions depend on whether you are running Teleport on the public
+internet, a local container, or a private network:
+
+<Tabs>
+
+  <TabItem label="Public internet deployment with Let's Encrypt">
+    Let's Encrypt verifies that you control the domain name of your Teleport cluster
+    by communicating with the HTTPS server listening on port 443 of your Teleport
+    Proxy Service.
+
+    You can configure the Teleport Proxy Service to complete the Let's Encrypt
+    verification process when it starts up.
+
+    On the host where you will start the Teleport Auth Service and Proxy Service,
+    run the following `teleport configure` command. Assign 
+    <Var name="teleport.example.com" /> to the
+    domain name of your Teleport cluster and <Var name="user@example.com" /> to
+    an email address used for notifications (you can use any domain):
+
+    ```code
+    $ sudo teleport configure -o file \
+        --acme --acme-email=<Var name="user@example.com" description="Your email address to register with Let's Encrypt for TLS certificates" /> \
+        --cluster-name=<Var name="teleport.example.com" description="The domain name of your Teleport cluster" />
+    ```
+
+    Port 443 on your Teleport Proxy Service host must allow traffic from all sources.
+  </TabItem>
+  <TabItem label="Docker container">
+
+  The Docker container you started while beginning this guide mounts the
+  `teleport-tls` directory in `/etc/`, including a TLS certificate and private
+  key for Teleport.
+
+  On the container, run the following `teleport configure` command: 
```

Comment:
How did `teleport` get installed in the container?

Reply:
This uses the same installation command as the guide tells you to run on a Linux server: https://github.com/gravitational/teleport/pull/55751/files#diff-941e5a3006f0201e2680fcd5b589f38a4cfd9ea113a82b2af9f9729ecc090dffR163-R167